Claroty/claroty-xdome/tests/test_expired_certificate.json

```
@@ -0,0 +1,74 @@
+        "dialect": "Claroty xDome",
+        "dialect_uuid": "9f47aa9f-52d7-4849-9462-cf7fc8bcd51a"
+      }
+    },
+    "message": "CEF:0|Claroty|Industrial Device Security Platform|1.0|11111111|Expired TLS Certificate was sent over TLS 1.2 from 1.2.3.4 (Server) to 5.6.7.8 (PC)|5|certificate_C=US certificate_CN=PRDVRZ1.prod.example.com certificate_L=None certificate_O=None certificate_OU=None certificate_ST=None certificate_expiry_date=20160526192415Z certificate_start_date=20150527192415Z certificate_thumbprint=AF:40:01:58:A8:3F:B2:AB:3C:7A:36:67:FD:65:FA:50:6C:67:A6:59 certificate_type=Server device_asset_id=BWMGLTVSTE device_assignees=[] device_category=IT device_connection_type_list=['Ethernet'] device_ip_list=['1.2.3.4'] device_labels=[] device_mac_list=['00:50:56:97:02:d4'] device_manufacturer=None device_model=None device_network_list=['Corporate'] device_note=None device_os=Windows Server 2012 R2 NT 6.3 device_site_name=Main device_subcategory=Servers device_type=Server device_uid=71b2bde6-370d-4a00-840d-bd828de48364 domain=nrldv001.prod.example.com dst_ip=5.6.7.8 dst_mac=00:50:56:94:b2:86 dst_port=59889 event_description=Expired TLS Certificate was sent over TLS 1.2 from 1.2.3.4 (Server) to 5.6.7.8 (PC) event_id=1111111 event_timestamp=2022-02-06T14:24:10.146720+00:00 event_type=Expired TLS Certificate geo_location=None msg_category=comm_event protocol=TLS 1.2 sender_id=NRLDV001 server_port=None src_ip=1.2.3.4 src_mac=e8:98:6d:ce:1f:10 src_port=8083"
+  },
+  "expected": {
+    "message": "CEF:0|Claroty|Industrial Device Security Platform|1.0|11111111|Expired TLS Certificate was sent over TLS 1.2 from 1.2.3.4 (Server) to 5.6.7.8 (PC)|5|certificate_C=US certificate_CN=PRDVRZ1.prod.example.com certificate_L=None certificate_O=None certificate_OU=None certificate_ST=None certificate_expiry_date=20160526192415Z certificate_start_date=20150527192415Z certificate_thumbprint=AF:40:01:58:A8:3F:B2:AB:3C:7A:36:67:FD:65:FA:50:6C:67:A6:59 certificate_type=Server device_asset_id=BWMGLTVSTE device_assignees=[] device_category=IT device_connection_type_list=['Ethernet'] device_ip_list=['1.2.3.4'] device_labels=[] device_mac_list=['00:50:56:97:02:d4'] device_manufacturer=None device_model=None device_network_list=['Corporate'] device_note=None device_os=Windows Server 2012 R2 NT 6.3 device_site_name=Main device_subcategory=Servers device_type=Server device_uid=71b2bde6-370d-4a00-840d-bd828de48364 domain=nrldv001.prod.example.com dst_ip=5.6.7.8 dst_mac=00:50:56:94:b2:86 dst_port=59889 event_description=Expired TLS Certificate was sent over TLS 1.2 from 1.2.3.4 (Server) to 5.6.7.8 (PC) event_id=1111111 event_timestamp=2022-02-06T14:24:10.146720+00:00 event_type=Expired TLS Certificate geo_location=None msg_category=comm_event protocol=TLS 1.2 sender_id=NRLDV001 server_port=None src_ip=1.2.3.4 src_mac=e8:98:6d:ce:1f:10 src_port=8083",
+    "event": {
+      "kind": "alert",
+      "category": ["authentication"],
+      "type": ["info"],
```

Can you change the parser to set the correct category and type for this event?
```suggestion
      "category": ["network"],
      "type": ["connection"],
```